shim.go

```diff
@@ -30,7 +30,32 @@ type ShimInfo struct {
 	Port  string
 }
 
-func startShim(pod *vc.Pod) (int, error) {
+func startShim(info interface{}) (int, error) {
+	switch obj := info.(type) {
+	case *vc.Pod:
+		return startShimFromPod(obj)
+	case *vc.Process:
+		return startShimFromProcess(obj)
+	default:
+		return -1, fmt.Errorf("Invalid interface, could not start the shim")
+	}
+}
+
+func startShimFromProcess(process *vc.Process) (int, error) {
+	cmd := exec.Cmd{
+		Path: defaultShimPath,
+		Args: []string{"-t", process.Token, "-s", defaultProxyIP, "-p", defaultProxyPort},
+		Env:  os.Environ(),
+	}
+
+	if err := cmd.Start(); err != nil {
+		return -1, err
+	}
+
+	return cmd.Process.Pid, nil
+}
+
```

Comment:
Ok, I think it would be cleaner if we could have a common function for both cases: ` func startShim(proxy vc.CCProxyConfig, process *vc.Process) (int, error)`

And then have a `startContainerShim(proxy vc.CCProxyConfig, container *vc.Container)` that would convert a `vc.Container` into its init `vc.Process`.

This means our `ShimInfo` structure and `getShimInfo()` routine can be removed.